src/main/resources/templates/footer.html

```diff
@@ -0,0 +1,41 @@
+<!doctype html>
+
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+    <title>Footer</title>
+
+</head>
+<body>
+<footer class="text-center text-white" style="background-color: rgba(0, 0, 0, 0.2);" th:fragment="footer">
+    <div class="container pt-4">
+        <section class="mb-4">
+            <!-- Facebook -->
+            <a href="https://google.com" type="button" class="btn btn-secondary">
+                <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor"
```

Comment:
Is there no other solution than writing these `svg` elements inside the markup?
Maybe you can have actual SVG files as static resources and reference them in the markup.

Reply:
Created SVG files for all the images and updated lines.